src/chain_sync/chain_muxer.rs

```diff
@@ -419,6 +419,10 @@ where
                 metrics::LIBP2P_MESSAGE_TOTAL
                     .with_label_values(&[metrics::values::PEER_DISCONNECTED])
                     .inc();
+                // Unset heavist tipset for unset peers
```

Comment:
why? what does it have to do with this subcommand?

Reply:
I was using this to cross-check the correctness of the metrics, it's part of https://github.com/ChainSafe/forest/issues/3218 and I can convert it into a separate PR if you like.

Reply:
Let's do it in a separate PR and comment on it some more, then.

Reply:
Removed